src/encode_data.rs

```diff
@@ -0,0 +1,4 @@
+#![allow(clippy::all)]
+use daedalus::gen_encode_data;
+
+gen_encode_data!("./Embedded-Base/cangen/can-messages");
```

Comment:
can you make the filepath a constant in like `lib.rs` or something so we dont forget to change it on encode but remeber decode if it ever needs to change?

Reply:
Adding this as a constant at the top of daedalus `lib.rs`